src/tiled/tiled.pro

```diff
@@ -141,6 +141,7 @@ SOURCES += aboutdialog.cpp \
     flexiblescrollbar.cpp \
     flipmapobjects.cpp \
     geometry.cpp \
+	gotodialog.cpp \
```

Comment:
You've used a tab here while the other lines are using 4 spaces. Please use 4 space as well (also applies to header file entry below).